ceph_deploy/rgw.py

```diff
@@ -174,13 +173,13 @@ def rgw_create(args):
                 msg = "exisiting rgw '%s:%s' has a different hostname" % (hostname, name)
                 LOG.error(msg)
                 raise RuntimeError(msg)
-
+        rgw_dns_name_value = "%s:%s" % (hostname, port)
         if cfg.has_option(enitity,'rgw_dns_name') is False:
-            cfg.set(enitity, 'rgw_dns_name', hostname)
+            cfg.set(enitity, 'rgw_dns_name', rgw_dns_name_value)
```

Comment:
@kmroz and me found that we shouldn't add the port to the dns name, it works properly with just the hostname, with port things will be broken